src/assets/zendesk-widget.js

```diff
@@ -13,3 +13,13 @@ window.zEmbed || function (e, t) {
     n && (this.domain = n), e.id = "js-iframe-async", e.src = "https://assets.zendesk.com/embeddable_framework/main.js", this.t = +new Date, this.zendeskHost = "prx.zendesk.com", this.zEQueue = a, this.body.appendChild(e)
   }, o.write('<body onload="document._l();">'), o.close()
 }();
+
+window.zESettings = {
+  webWidget: {
+    // https://developer.zendesk.com/embeddables/docs/widget/api
+    // is possible to set positioning and offset to top right but cannot configure button padding
```

Comment:
I messed around with putting this in the header per the mockup, but I found that you cannot configure the padding on the button. I found that I had to change the button position with the changing size of the header with those responsive settings. Our media query didn't match up to what they were doing to determine mobile layouts, so I couldn't use offset to handle this positioning. It all just ended up feeling like a bad idea.

Reply:
I'm also just noticing that we don't have the feedback widget in Publish.  Feels like it should be there, right?